base/windows/CharsStatusWindow.js

```diff
@@ -74,47 +74,53 @@ export class CharsStatusWindow {
             info_sprites_obj.hp_bar_graphics.beginFill(STATUS_BAR_COLOR_GOOD, 1);
             info_sprites_obj.hp_bar_graphics.drawRect(0, 0, STATUS_BAR_WIDTH, STATUS_BAR_HEIGHT);
             info_sprites_obj.hp_bar_graphics.endFill();
-            this.status_window.group.add(info_sprites_obj.hp_bar_graphics);
+            info_sprites_obj.group.add(info_sprites_obj.hp_bar_graphics);
 
             info_sprites_obj.hp_bar_damage_graphics = this.game.add.graphics(0, 0);
             info_sprites_obj.hp_bar_damage_graphics.default_y = y_pos_bar;
-            this.status_window.group.add(info_sprites_obj.hp_bar_damage_graphics);
+            info_sprites_obj.group.add(info_sprites_obj.hp_bar_damage_graphics);
 
-            info_sprites_obj.hp_header = this.status_window.set_text_in_position("HP", base_x_pos, y_pos);
+            info_sprites_obj.hp_header = this.status_window.set_text_in_position("HP", base_x_pos, y_pos, false, false, this.status_window.font_color, false, group_key);
             const x_number_pos = base_x_pos + STAT_X;
-            info_sprites_obj.hp = this.status_window.set_text_in_position(char.current_hp.toString(), x_number_pos, y_pos, true);
+            info_sprites_obj.hp = this.status_window.set_text_in_position(char.current_hp.toString(), x_number_pos, y_pos, true, false, this.status_window.font_color, false, group_key);
 
             y_pos = this.name_y + 2 * numbers.FONT_SIZE;
             y_pos_bar = y_pos + numbers.FONT_SIZE - STATUS_BAR_HEIGHT;
             info_sprites_obj.pp_bar_graphics = this.game.add.graphics(base_x_pos, y_pos_bar);
             info_sprites_obj.pp_bar_graphics.beginFill(STATUS_BAR_COLOR_GOOD, 1);
             info_sprites_obj.pp_bar_graphics.drawRect(0, 0, STATUS_BAR_WIDTH, STATUS_BAR_HEIGHT);
             info_sprites_obj.pp_bar_graphics.endFill();
-            this.status_window.group.add(info_sprites_obj.pp_bar_graphics);
+            info_sprites_obj.group.add(info_sprites_obj.pp_bar_graphics);
 
             info_sprites_obj.pp_bar_damage_graphics = this.game.add.graphics(0, 0);
             info_sprites_obj.pp_bar_damage_graphics.default_y = y_pos_bar;
-            this.status_window.group.add(info_sprites_obj.pp_bar_damage_graphics);
+            info_sprites_obj.group.add(info_sprites_obj.pp_bar_damage_graphics);
+
+            info_sprites_obj.pp_header = this.status_window.set_text_in_position("PP", base_x_pos, y_pos, false, false, this.status_window.font_color, false, group_key);
+            info_sprites_obj.pp = this.status_window.set_text_in_position(char.current_pp.toString(), x_number_pos, y_pos, true, false, this.status_window.font_color, false, group_key);
 
-            info_sprites_obj.pp_header = this.status_window.set_text_in_position("PP", base_x_pos, y_pos);
-            info_sprites_obj.pp = this.status_window.set_text_in_position(char.current_pp.toString(), x_number_pos, y_pos, true);
+            info_sprites_obj.group.visible = true;
```

Comment:
I think we don't need this line.

Reply:
True, I just converted the old one. I will remove it